setup.py

```diff
@@ -41,7 +41,7 @@
     install_requires=[
         "Django>=3.1,<3.3",
         "Wagtail>=2.14,<2.16",
-        "lxml==4.6.3",
+        "lxml>=4.6,<4.7",
```

Comment:
**Issue**: This requirement still permits 4.6.3. We should insist on `>=4.6.5`.

**Question**: Why `<4.7`? Version 4.7.1 released yesterday doesn't claim to introduce any breaking changes. https://lxml.de/4.7/changes-4.7.1.html

Reply:
Thank Nick. How about if we do `"lxml>=4.7,<4.8"`

Reply:
Yep, that works for versions existing as we speak.

Reply:
Thanks Nick i updated the version constraints.